assets/src/block-editor/QueryBlockExtension/QueryBlockExtension.js

```diff
@@ -0,0 +1,125 @@
+import {RadioControl, PanelBody} from '@wordpress/components';
+import {InspectorControls} from '@wordpress/block-editor';
+import {useCallback, useEffect, useMemo, useState} from '@wordpress/element';
+
+const {addFilter} = wp.hooks;
+const {__} = wp.i18n;
+
+const targetBlocks = ['core/post-template'];
+
+const layoutTypes = [
```

Comment:
This layout option `{label: 'List', value: 'default', columnCount: 3}` might be enabled within Posts List whenever we work on [PLANET-7416](https://jira.greenpeace.org/browse/PLANET-7416)